src/tests/mod.rs

```diff
@@ -89,7 +89,10 @@ fn test_rpm_header_base(package: RPMPackage) -> Result<(), Box<dyn std::error::E
     );
     assert_eq!(package.metadata.get_build_time().unwrap(), 1540945151);
 
-    assert_eq!(package.metadata.get_payload_compressor().unwrap(), "xz");
+    assert!(matches!(
+        package.metadata.get_payload_compressor(),
+        Err(RPMError::UnknownCompressorType(_))
```

Comment:
Here and below you should match `Err(RPMError::UnknownCompressorType("xz".to_string()))` instead

Reply:
Done